projects/ngx-kjua/package.json

```diff
@@ -27,7 +27,8 @@
   "peerDependencies": {
     "@angular/common": ">=14.0.0",
     "@angular/core": ">=14.0.0",
-    "kjua-svg": "^1.13.1"
+    "kjua-svg": "^1.13.1",
```

Comment:
hm I think we can remove kjua-svg then?

Reply:
@werthdavid Absolutely, good catch.

It's done